paper/paper.md

```diff
@@ -0,0 +1,233 @@
+---
+title: 'TransitionInTimeSeries.jl: A performant, extensible and reliable software for
+reproducible detection and prediction of transitions in timeseries'
+tags:
+  - Julia
+  - nonlinear dynamics
+  - timeseries analysis
+  - change point detection
+  - resilience loss
+  - critical slowing down
+  - early warning signals
+authors:
+  - name: Jan Swierczek-Jereczek
+    orcid: 0000-0003-2213-0423
+    affiliation: "1, 2"
+  - name: George Datseris
+    orcid: 0000-0003-0872-7098
+    affiliation: 3
+affiliations:
+ - name: Department of Earth Physics and Astrophysics, Complutense University of Madrid.
+   index: 1
+ - name: Geosciences Institute, CSIC-UCM.
+   index: 2
+ - name: University of Exeter.
+   index: 3
+date: 13 February 2024
+bibliography: paper.bib
+---
+
+# Summary
+
+Changes in the regime of a nonlinear dynamical system can significantly impact individuals
+and society. Examples of this are ubiquitous and include the onset of cardiac arythmia
+[@tse_mechanisms_2016], the deglaciation of Earth about 20,000 years ago [@wolff_changes_2010]
+and the recent price collapse of many cryptocurrencies [@ismail_detecting_2020]. The systems
+displaying such dramatic events are therefore usually monitored by measuring state variables
+that are believed to be representative of the underlying process. The resulting timeseries
+can be analysed with a wide range of generic techniques to detect past events and predict
+future ones. 
+
+# Statement of need
+
+Over the last decades, methods to detect and predict transitions from timeseries have gained
+a lot of attention, both inside and outside of the scientific community. For instance, recent
+work predicting a collapse of the Atlantic Meridional Overturning Circulation between 2025
+and 2095 has led to no less than 870 news outlets and 4100 tweets [@ditlevsen_warning_2023],
+largely because of the substantial implications of such a collapse for human societies.
+However, some published work is not reproducible despite their strong policy implications.
```

Comment:
This paper provides open-source R code that should be reproducible, but I did not check myself. I mention this one because it had the biggest impact in popular media to my knowledge.

Reply:
Sure, but the problem is that we are making a definitive statement: "some published work is not reproducible despite ..." . How do we substantiate this in the scientific sense?

We can argue instead: "A common concern in the scientific community is that published work on the topic is difficult to reproduce, despite the impact it implies for humanity." which doesn't assert a definitive statement, but would still be acceptable by pretty much everyone.